package.json

```diff
@@ -33,7 +33,7 @@
     "polymer-analyzer": "2.0.0-alpha.40",
     "polymer-bundler": "2.0.0-pre.14",
     "polymer-project-config": "^2.0.0",
-    "sw-precache": "^4.2.0",
+    "sw-precache": "^5.0.0",
```

Comment:
^5.0.0 will get 5.1.1 and higher 5's, but it looks like there are some bug fixes in those later versions that we specifically want to include. +1 on changing this to `^5.1.1`